applications/luci-app-mwan3/htdocs/luci-static/resources/view/mwan3/mwan3.js

```diff
@@ -0,0 +1,95 @@
+function secondsToString(time) {
+	var seconds = parseInt(time, 10);
+
+	var hrs = Math.floor(seconds / 3600);
+	seconds -= hrs*3600;
+	var mnts = Math.floor(seconds / 60);
+	seconds -= mnts*60;
+	return String.format("%sh:%sm:%ss", hrs, mnts, seconds);
+}
+
+function renderMwan3Status(status) {
+
+	if (status.interfaces) {
+		var statusview = '';
+		for ( var iface in status.interfaces) {
+			var state = '';
+			var css = '';
+			var time = '';
+			switch (status.interfaces[iface].status) {
+				case 'online':
+					state = _('Online');
+					time = String.format(
+						'<div><strong>%s:&nbsp;</strong>%s</div>',
```

Comment:
I suggest to replace every `%s` with `%h` which takes care of HTML escaping any special characters in the input. Granted, it is unlikely that the backend status reply values contain `<`, `>`, `&` etc. but you can't fully rule it out (invalid user config, ...).

Since we already had "stored XSS" reports for these kinds of invalid config value issues in the past, it is better to avoid it right away.